src/home/views/index.tsx

```diff
@@ -24,28 +27,86 @@ const HomeSection = () => {
   const noChannel = !channel && typeof channel !== "undefined";
 
   const userPermissions = user?.userPermissions || [];
+  const hasPermissionToManageOrders = hasPermissions(userPermissions, [
+    PermissionEnum.MANAGE_ORDERS,
+  ]);
+  const hasPermissionToManageProducts = hasPermissions(userPermissions, [
+    PermissionEnum.MANAGE_PRODUCTS,
+  ]);
 
-  const { data } = useHomeQuery({
-    displayLoader: true,
+  const {
+    data: homeActivities,
+    loading: homeActivitiesLoading,
+    error: homeActivitiesError,
+  } = useHomeActivitiesQuery({
+    skip: noChannel || !hasPermissionToManageOrders,
+  });
+
+  const {
+    data: homeTopProducts,
+    loading: homeTopProductsLoading,
+    error: homeTopProductsError,
+  } = useHomeTopProductsQuery({
+    skip: noChannel || !hasPermissionToManageProducts,
+    variables: {
+      channel: channel?.slug,
+    },
+  });
+
+  const {
+    data: homeNotificationsData,
+    loading: homeNotificationsLoaing,
+    error: homeNotificationsError,
+  } = useHomeNotificationsQuery({
     skip: noChannel,
+    variables: {
+      channel: channel?.slug,
+      hasPermissionToManageOrders,
+    },
+  });
+
+  const {
+    data: homeAnaliticsData,
+    loading: homeAnaliticsLoading,
+    error: homeAnaliticsError,
+  } = useHomeAnaliticsQuery({
+    skip: noChannel || !hasPermissionToManageOrders,
```

Comment:
could you co-locate it in the the components that they are related to? Having all calls within some parent component causes multiple renders whenever any of calls trigger some state update